src/python/turicreate/toolkits/_model.py

```diff
@@ -27,6 +27,16 @@
 
 MODEL_NAME_MAP = {}
 
+# Object detector use C++ codepath
+od_use_cpp = True
+if not os.environ.has_key('od_use_cpp') or os.environ.get('od_use_cpp')=="0":
+    od_use_cpp = False
+
+# Activity Classifier use C++ codepath
+ac_use_cpp = True
+if not os.environ.has_key('ac_use_cpp') or os.environ.get('ac_use_cpp')=="0":
+    ac_use_cpp = False
+
```

Comment:
I suspect these should be encapsulated into helper functions, instead of code running at the top level on import. Probably this should be handled by https://github.com/apple/turicreate/pull/2467
?

Reply:
@nickjong That's a good idea. I'm thinking of adding a helper function to `_internal_utils.py`. Does that sound good?